lib/assessment.js

```diff
@@ -13,14 +21,23 @@ var min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  var results = [];
+  for (var i = 0; i < results.length; i++) {
+    total += results[i]
+  }
+  return total;
 };
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
+// COME BACK
 var newArray = function newArray(length, defaultValue) {
-
+  var ary = [];
+  for (var i = 0; i < ary.length; i++){
+    ary[] = defaultValue(i);
```

Comment:
Since defaultValue isn't a function, this won't work. 

Additionally, `ary[] = ` is not the right syntax. To add something to an array, you can use `.push()`, or you can specify an index and a value, i.e. `ary[3] = "blah"`.